lib/hash_ring/ets.ex

```diff
@@ -252,21 +288,47 @@ defmodule ExHashRing.HashRing.ETS do
       {:reply, {:error, :node_exists}, state}
     else
       nodes = [{node_name, num_replicas} | nodes]
-      {:reply, {:ok, nodes}, rebuild(%{state | nodes: nodes})}
+      {:reply, {:ok, nodes}, update_nodes(state, nodes)}
+    end
+  end
+
+  def handle_call({:add_nodes, nodes}, _from, %__MODULE__{} = state) do
```

Comment:
Are you suggesting just in the handle_call functions, because that seems fine.  I think the add_node/2 and remove_node/2 functions are nice for the client.

Reply:
Implemented add_node/2 and remove_node/2 in terms of :add_nodes and :remove_nodes in https://github.com/discord/ex_hash_ring/pull/10/commits/173fbda6fff6a11075af73ad466eed548991667b